tests/cli/test_cmds_mark_for_deployment.py

```diff
@@ -36,6 +37,12 @@ class fake_args:
     timeout = 10.0
 
 
+@fixture(autouse=True, scope='session')
```

Comment:
`autouse=True, scope='session' ` makes me feel dirty:

 - these four lines at the top of the file magically apply to tests at the bottom of the file (300+ lines away).
- doesn't this make it impossible to test `get_authors_to_be_notified`?
- `scope='session'` will mean that the mock will never be reset - so if someone modifies the return value it won't be undone for the next test.

In general, I prefer tests to be independent more than I like DRY test code. Specifically, I like to be able to know exactly what a test is doing just by looking at that test; not having to scan the whole file (or any plugins that get installed) to know what might be mocked out.

Reply:
Ok